data/campaigns/Eastern_Invasion/scenarios/11_Captured.cfg

```diff
@@ -125,6 +125,7 @@
         color=white
         type=Orcish Warlord
         id=Varrak-Klar
+        #po: In S11, an orc with so much gold for recruitment that the player can't beat them, acts as a turn limit.
```

Comment:
How is this at all relevant to the translation of this guy's name? I guess it can never hurt to have more information, but… I can't imagine any case where this would be useful, unless the translator is trying to be humourous.

Reply:
The name of a unit also gets an automated collection of other information (type, gender, possibly which scenarios they appear in), so it seems the natural point to collect other information.

My intention here was to write a 3-line paragraph in the .po file, which hopefully makes sense when reading the output that's in the .po files instead of reading it split in the .cfg files; I was trying to document how he's affected by the renaming mechanism. Too much detail admittedly, and it wasn't needed for Rakkha at all.

> In S11, an orc with so much gold for recruitment that the player can't beat them, acts as a turn limit.
> In S12, a leader with a high but not unbeatable amount of gold.
> However, if Chief Dra-Nak survived S11 then S12's Varrak-Klar will be changed to an Orcish Sovereign and renamed to appear to be the Chief pursuing the escapees.